app/0.14.x/logging.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Logging Reference
+---
+
+# Removing Certain Elements From Your Kong Logs
+
+With new regulations surrounding protecting private data like GDPR, there is a chance you may need to change your logging habits. If you use Kong as your API Gateway, this can be done in a single location to take effect on all of your APIs. This guide will walk you through one approach to accomplishing this, but there are always different approaches for different needs.
+
+For this example, let’s say you want to remove any instances of an email address from your kong logs. The emails addresses may come through in different ways, for example something like `/apiname/v2/verify/alice@example.com` or `/v3/verify?alice@example.com`. In order to keep these from being added to the logs, we will need to use a custom NGINX template.
+
+To start using a custom NGINX template, first get a copy of our template. This can be found [https://docs.konghq.com/0.13.x/configuration/#custom-nginx-configuration](https://docs.konghq.com/0.13.x/configuration/#custom-nginx-configuration) or copied from below
+```
+# ---------------------
+# custom_nginx.template
+# ---------------------
+ 
+worker_processes ${{NGINX_WORKER_PROCESSES}}; # can be set by kong.conf
+daemon ${{NGINX_DAEMON}};                     # can be set by kong.conf
+ 
+pid pids/nginx.pid;                      # this setting is mandatory
+error_log logs/error.log ${{LOG_LEVEL}}; # can be set by kong.conf
+ 
+events {
+    use epoll; # custom setting
+    multi_accept on;
+}
+ 
+http {
+    # include default Kong Nginx config
+    include 'nginx-kong.conf';
+ 
+    # custom server
+    server {
+        listen 8888;
+        server_name custom_server;
+ 
+        location / {
+          ... # etc
+        }
+    }
+}
+```
+
+In order to control what is placed in the logs, we will be using the NGINX map module in our template. For more detailed information abut using the map directive, please see this guide . This will create a new variable whose value depends on values of one or more of the source variables specified in the first parameter. The format is:
```

Comment:
"this guide" should link to something?

Reply:
yup, it sure should. Fixed in https://github.com/Kong/docs.konghq.com/pull/820